packages/templates/tests.spec.ts

```diff
@@ -0,0 +1,106 @@
+import fs from "fs";
+import path from "path";
+import { exec } from "child_process";
+
+jest.setTimeout(600000);
+
+describe("Templates", () => {
+  const rootDir = __dirname;
+
+  // Mapping of all project types (app, plugin, etc) to all supported languages
+  const projectLanguages: Record<string, string[]> = {};
+
+  // Define the commands to run for each language
+  const languageTestCommands: Record<string, Record<string, string>> = {
+    typescript: { build: "yarn build", test: "yarn test" },
+    "typescript-node": {
+      // Uncomment when the helloworld wrapper has been deployed to polywrap.eth
```

Comment:
I don't think we need it deployed to polywrap.eth, we can just redirect it to the current uri. (i.e. ens/goerli/hello.as.demos.wraplib.eth)

Reply:
@Niraj-Kamdar Goerli was giving me some trouble, so I decided to use the hello-world wrapper deployed on `mainnet` via @nerfZael's wrappers.io.
Address is `ens/helloworld.polytest.eth`.